src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs

```diff
@@ -76,5 +77,26 @@ public async Task<IActionResult> AddDraftapprenticeships(BulkUploadAddDraftAppre
             }
             return Ok(result);
         }
+
+        [HttpPost]
+        [Route("AddLog")]
+        public async Task<IActionResult> AddLog(BulkUploadAddLogRequest request)
+        {
+            var result = await _mediator.Send(
+                new BulkUploadAddLogCommand
+                {
+                    ProviderId = request.ProviderId,
+                    FileName = request.FileName,
+                    RplCount = request.RplCount,
+                    RowCount = request.RowCount,
+                    FileContent = request.FileContent
+                });
+
+            if (result == null)
+            {
+                return NotFound();
+            }
```

Comment:
Do we need the not found result?  We don't have a test covering it and it shouldn't happen. Should it?

Reply:
Good shout. Removed